### ISSUE TITLE
deduplication with UMI

### ISSUE THREAD

**Issue by yupingz** [OpenGene/fastp#457]
I could not find description about how deduplication is done for fastq with UMI. I found "fastp considers one read as duplicated only if its all base pairs are identical as another one." Is it the same for fastq with UMI? Thanks!

**Comment by sfchen** [comment]
Yes, UMIs are taken  into account. Dedup is  performed before UMIs are removed.

**Comment by annajbott** [comment]
Brilliant thanks!
